content/md/first-steps/manage-product-identifiers.md

```diff
@@ -0,0 +1,97 @@
+---
+id: manage-product-identifiers
+themes: catalog-settings
+title: Manage your product identifiers
+popular: true
+ee: true
+ge: true
+related: manage-your-attributes
+---
+
+You might need to identify products in the PIM for various reasons: Search for a specific product, update one or more products, or even synchronize product data with other systems.
+
+# How to identify a product?
+
+SKU used to be the only product identifier in the PIM.
+There are now 2 ways to identify a product:
+1. Use the identifier attribute called SKU
+1. Use the product UUID, which is the most future-proof method
+
+# Manage your products with the Identifier attribute
+
+## Description of the Identifier attribute
+
+The identifier is an attribute with a few specific characteristics:
+- Right now, only one identifier attribute is created by default. It is always called “SKU” as the attribute code can't be modified. Only its label can be modified.
+- The value of this attribute must be unique per product
+- This attribute is not localizable (per locale) nor scopable (per channel)
+
+::: warning
+Starting in September, the identifier attribute will evolve:
```

Comment:
Do not mention the month in this page, as it will be not the same september in one year. I can bet that you'll forget to remove this 'September'. My advide is to make this document timeless.

Reply:
I agree with Alexandre, but I understand the need for this specific use case as this documentation will be updated when the SKU will become optional, right?

Reply:
Yes exactly, he's aware of that!